inputs/BaseInput.php

```diff
@@ -13,6 +13,18 @@
  */
 abstract class BaseInput
 {
+    /**
+     * Add options
+     * Allows to add options to the variable $options in the gameoflife.php file.
+     * @param Getopt $_options
+     */
     abstract public function addOptions(Getopt &$_options);
+
+    /**
+     * Fill field
+     * Fills the board with the wished number of cells. The filling will be set via parameter.
```

Comment:
What do you mean with „The filling will be set via parameter“?

Reply:
Please answer the question and do not just mark this as resolved without answering it. 😬

Reply:
Well, I made a little mistake and wrote the documentation of the fillField function (in Random.php) into the BaseInput class, accidentally. I've already corrected it, which is why I didn't answer the question 2 days ago.

Reply:
Ok, that was to implicit to notice for me, sorry.